### PR TITLE
topview should not using parent of mainwindow

### LINKED ISSUE #2007: MSUI with Windows: TopView window is always on top of MSUI main window
Under Windows 10, MSUI TopView window is always on top of MSUI main window. In addition, the TopView is not appearing in the Windows Task bar (other views are visible there). Alt +Tab also does not allow to change TopView windows.

grafik

Further, "Server/Layer" window is also always on top of corresponding TopView/SideView

### PR DESCRIPTION
**Pull request by ReimarBauer**
**Purpose of PR?**:

Fixes #2007

**Does this PR introduce a breaking change?**
API change

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Additional information for reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Does this PR results in some Documentation changes?**
_If yes, include the list of Documentation changes_

**Checklist:**
- [X] Bug fix. Fixes #<issue number>
- [ ] New feature (Non-API breaking changes that adds functionality)
- [ ] PR Title follows the convention of  `<type>: <subject>`
- [ ] Commit has unit tests

### CODE REVIEW

**Comment by ReimarBauer** [comment]
@joernu76 the tests got blocked by a similiar expensive use of slots as you fixed in the kml docking widget.

(Of course speed is much faster now too)

File: mslib/msui/topview.py [hunk 1]
```diff
@@ -185,13 +185,27 @@ class MSUITopViewWindow(MSUIMplViewWindow, ui.Ui_TopViewWindow):
...
 
-    def __init__(self, parent=None, model=None, _id=None, active_flighttrack=None, mscolab_server_url=None, token=None):
+    def __init__(self, parent=None, mainwindow=None, model=None, _id=None,
+                 active_flighttrack=None, mscolab_server_url=None, token=None):
         """
         Set up user interface, connect signal/slots.
         """
         super().__init__(parent, model, _id)
         logging.debug(_id)
-        self.ui = parent
```
Comment (joernu76) [inline mslib/msui/topview.py:194]: wouldn't a reference to mainwindow not "simpler" here?

**Review by joernu76 — APPROVED** [review]
I think the same problem affects Side&Linearview? Still a step in the right direction.

File: mslib/msui/topview.py (hunk 1 above)
Comment (ReimarBauer) [reply mslib/msui/topview.py:194]: Later we use parent when the docking widget is called from topview.

That shows currently what we need.

**Comment by ReimarBauer** [comment]
> I think the same problem affects Side&Linearview? Still a step in the right direction.

Sideview and Linearview don't have multiple flightpath dockingwidget. By this the problem was introduced.

**Pull request merged**